src/features/wallet/helpers/ERC20TransferButton.tsx

```diff
@@ -0,0 +1,43 @@
+// Note: This only exists for testing. Renders a button to transfer UMA for testing.
+import { FC } from "react";
+import Button from "common/components/button";
+import { ethers } from "ethers";
+import createERC20ContractInstance from "common/utils/web3/createERC20ContractInstance";
+
+interface Props {
+  network: ethers.providers.Network | null;
+  signer: ethers.Signer | null;
+  votingAddress: string | null;
+  hotAddress: string | null;
+}
+const ERC20TransferButton: FC<Props> = ({ network, signer, votingAddress }) => {
```

Comment:
i would probably rename signer and votingAddress to fromAddress and toAddress, or something, since this utility shouldnt really care about the context of the addresses

Reply:
+1